src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -133,6 +137,7 @@ public without sharing class UTIL_OrgTelemetry_SVC {
             // operation.
             when NPSP_Settings {
                 handleNPSPSettings();
+                handleCustomInstallmentSettings();
```

Comment:
Because the `handleCustomInstallmentSettings` method has a query against the RD object which could (though unlikely) timeout, it might be better to separate this action into a different `TelemetryBatchCategory`.